Makefile.am

```
@@ -41,6 +41,7 @@ DIST_DOCS = $(wildcard doc/*.md) $(wildcard doc/release-notes/*.md)
 DIST_CONTRIB = $(top_srcdir)/contrib/bitcoin-cli.bash-completion \
 	       $(top_srcdir)/contrib/bitcoin-tx.bash-completion \
 	       $(top_srcdir)/contrib/bitcoind.bash-completion \
+	       $(top_srcdir)/contrib/debian \
```

```suggestion
	       $(top_srcdir)/contrib/debian/copyright \
```

Since that is the only file left. And explicitly naming it prevents us from accidentally including files with an unsuitable license as pointed out in https://github.com/bitcoin/bitcoin/pull/12051#issuecomment-354538415